test/common/upstream/load_balancer_fuzz.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "envoy/config/cluster/v3/cluster.proto";
+import "google/protobuf/empty.proto";
+
+message UpdateHealthFlags {
+  //The boolean corresponds to whether this action updates the failover set or not
+  bool failover_host_set = 1;
+  uint32 num_healthy_hosts = 2;
```

Comment:
Why just number? I think doing a proper subset selection is more useful, since for some LBs, order will matter. E.g. if you are only marking unhealthy and excluding those hosts that are at the front of the list, that's different than those at the end of a list in something like EDF.

Reply:
Changed algorithm to do proper subset selection.